marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/GetProjectAsZipTest.java

```diff
@@ -144,6 +146,20 @@ private void readZipProject() throws IOException {
         }
     }
 
+    private void loadPropertiesFromZipEntry(ZipFile zip, ZipEntry entry, Properties props) {
+        try {
+            InputStream ins = zip.getInputStream(entry);
+            if (ins == null) {
+                logger.warn("Received null InputStream for zip entry: " + entry.getName() + "; will ignore");
+                return;
+            }
+            props.load(zip.getInputStream(entry));
```

Comment:
Can we use the InputStream ins which we know is not null here?
`props.load(ins);`

Reply:
Thanks, forgot to change that.